### PR TITLE
Update profile greenearth 3059

### LINKED ISSUE #3059: Update Project Profile: GreenEarthOS
### Overview
As Admin team I want the Green Earth project's details to be up to date on the HackforLA website so that people don't try to join it.

### Action Items
- [x] Observe the existing syntax of the [front matter block](https://jekyllrb.com/docs/front-matter/) inside of the [green-earth-os.md](https://github.com/hackforla/website/blob/gh-pages/_projects/green-earth-os.md) file and find the `leadership` variable.
- [x] Remove everyone from the `leadership` variable
- [x] Add the following `technology` variable (syntax and spacing are important)
```
technologies: 
  - mkdocs
```
- [x] Add the following `tools` variable and value
```
tools: Google sheets, Google forms, Figma, FigJam
```
- [x] Update the project `status` from `Active` to `On Hold`

### Resources
- https://github.com/hackforla/website/blob/gh-pages/_projects/green-earth-os.md
- [HFLA wiki on project.md files](https://github.com/hackforla/website/wiki/project.md-file-template)
- [Jekyll Docs on front matter](https://jekyllrb.com/docs/front-matter/)

### PR DESCRIPTION
**Pull request by frankstepanski**
Fixes #3059 

### What changes did you make and why did you make them?

  -  Updated project details for GreenEarth
  
  https://github.com/hackforla/website/blob/gh-pages/_projects/green-earth-os.md

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

before

</details>

<details>
<summary>Visuals after changes are applied</summary>

after

</details>

### CODE REVIEW

**Comment by Jaretzbalba** [comment]
ETA: End of the day on 6/01
Availability: 2 hours

File: _projects/green-earth-os.md [hunk 1]
```diff
@@ -13,36 +13,23 @@ leadership:
...
-    picture: https://avatars.githubusercontent.com/JessicaFB
-  - name: María-Inés Montes
-    role: UX/UI Designer
-    links:
-      slack: 'https://hackforla.slack.com/team/U027NG94TQB'
-      github: 'https://github.com/maria-ines-montes'
-    picture: https://avatars.githubusercontent.com/maria-ines-montes                    
+      slack: 'https://hackforla.slack.com/team/U02R9MKA6KH'
+      github: 'https://github.com/rpradheap'
+    picture: https://avatars.githubusercontent.com/samaynathani
```
Comment (Jaretzbalba) [inline _projects/green-earth-os.md:32]: The GitHub handle for Raji needs to be updated as it is currently linked to Samay Nathani's profile.

**Review by Jaretzbalba — CHANGES_REQUESTED** [review]
@frankstepanski Thank you for working on this issue! Based on what's requested in the issue for this PR, please add a technologies section to the md file as well. I also left a comment regarding a link you need to update as well as a link to a profile picture for Sanya. Thanks again!

File: _projects/green-earth-os.md [hunk 2]
```diff
@@ -13,36 +13,23 @@ leadership:
...
     picture: https://avatars.githubusercontent.com/ExperimentsInHonesty
-  - name: Akila Sankaran
+  - name: Sanya Nijhawan
     role: Product Manager
     links:
-      slack: 'https://hackforla.slack.com/team/U02462CCJK0'
-      github: 'https://github.com/akila-git'
-    picture: https://avatars.githubusercontent.com/akila-git
+      slack: 'https://hackforla.slack.com/team/U031FLYS561'
+      github: 'https://github.com/sanya301'
```
Comment (Jaretzbalba) [inline _projects/green-earth-os.md:20]: Sanya needs a link for her profile picture here

**Comment by luke-karis** [comment]
EDIT: Sorry this has taken me so long, I will definitely review it on June 6
ETA: End of day June 6
Availability 2 hours

**Review by luke-karis — CHANGES_REQUESTED** [review]
Hey @frankstepanski great job working on this issue! I agree with basically all of Jaret's comments we should try to get a picture for Sanya and change the picture for Raji. While the ticket says we should also add a technologies section for MkDocs I feel that we could maybe just throw this in the Tools section. What do you think @Jaretzbalba? Maybe we should check in with the creator of the ticket.

**Comment by blulady** [comment]
Availability: 1 hour
ETA: 7/1/22 (waiting for review request)

**Comment by blulady** [comment]
@frankstepanski Please give us your update by August 15 or we will assume that you are no longer working on this issue and close it.

**Comment by frankstepanski** [comment]
Sorry, my mother passed 2 months ago. 
ETA: By Wednesday (8/17)

**Pull request closed without merging**